front/src/Phaser/UserInput/UserInputManager.ts

```diff
@@ -186,6 +185,11 @@ export class UserInputManager {
         this.initKeyBoardEvent();
         this.isInputDisabled = false;
     }
+
+    isControlsEnable() {
+        return !this.isInputDisabled;
+    }
```

Comment:
Maybe we could use a getter here?

```
    get isControlsEnabled() {
        return !this.isInputDisabled;
    }
```

Also, `isControlsEnabled` rather than `isControlsEnable`